Documentation/ceph-fs-subvolumegroup.md

```diff
@@ -0,0 +1,44 @@
+---
+title: CephFilesystemSubVolumeGroup CRD
+weight: 3610
+indent: true
+---
+
+{% include_relative branch.liquid %}
+
+This guide assumes you have created a Rook cluster as explained in the main [Quickstart guide](quickstart.md)
+
+# CephFilesystemSubVolumeGroup CRD
+
+Rook allows creation of Ceph Filesystem [SubVolumeGroups](https://docs.ceph.com/en/latest/cephfs/fs-volumes/#fs-subvolume-groups) through the custom resource definitions (CRDs).
+Filesystem subvolume groups are an abstraction for a directory level higher than Filesystem subvolumes to effect policies (e.g., File layouts) across a set of subvolumes.
+For more information about CephFS volume, subvolumegroup and subvolume refer to the [Ceph docs](https://docs.ceph.com/en/latest/cephfs/fs-volumes/#fs-volumes-and-subvolumes).
+
+## Creating daemon
+
+To get you started, here is a simple example of a CRD to create a subvolumegroup on the CephFilesystem "myfs".
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephFilesystemSubVolumeGroup
+metadata:
+  name: my-subvolumegroup
+  namespace: rook-ceph # namespace:cluster
+spec:
+  # volumeName is the name of the CephFilesystem CR
+  volumeName: myfs
```

Comment:
Since it refers to the Filesystem CR, should we not call it filesystem?
```suggestion
  filesystem: myfs
```

Reply:
I was hesitant since the official ceph doc refers to it as `volumeName` but filesystem probably makes more sense. Unless we have multiple volume per fs...

Reply:
I dug a bit more and currently a volume in CephFS is a filesystem, when deploying a CephFilesystem we get one. So no multiple volumes.